src/components/Pressable.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { GestureObjects as Gesture } from '../handlers/gestures/gestureObjects';
```

Comment:
And just so it's not lost to time, I think the default config for ripple differs between this implementation and RN's one - here it's enabled by default.

Reply:
I disabled this in the latest commit, although I'm not sure if setting `rippleColor` to `transparent` is a good way of doing so 😅 

There don't seem to be any dedicated methods for disabling it on `RNButton`, and disabling the `Native` gesture had no effect either.

Reply:
Sounds fairly reasonable to me 😄